apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.js

```diff
@@ -82,26 +91,16 @@ router.post(
 
     if (courseOwners.length > 0) {
       // If the course already exists, display an error message containing the owners.
-      let error_message = `<p>The requested course (${short_name}) already exists.  Please contact the owner(s) of that course to request access to it.</p>`;
-      let formatted_owners = [];
-      courseOwners.forEach((c) => {
-        if (c.name !== null && c.uid !== null) {
-          formatted_owners.push(`${c.name} (<code>${c.uid}</code>)`);
-        } else if (c.name !== null) {
-          formatted_owners.push(c.name);
-        } else if (c.uid !== null) {
-          formatted_owners.push(`<code> ${c.uid} </code>`);
-        }
-      });
-      if (formatted_owners.length > 0) {
-        error_message += '<ul>';
-        formatted_owners.forEach((o) => {
-          error_message += '<li>' + o + '</li>';
-        });
-        error_message += '</ul>';
```

Comment:
Are you removing this on purpose?

Reply:
Sort of; this is the last case I know of where we were responding to a `POST` without a redirect, so no longer rendering a nice list in the response is definitely intentional. I suppose I could add the owner UIDs to the flash message?

That said: I think there are problems with the approach of detecting duplicates based solely on course short name (multiple universities could have a CS 101 course). I'm wondering if we should get rid of this check entirely? I raised this for discussion with Matt and Mariana.

Reply:
This might be a privacy issue too. This form is open to anyone, maybe exposing owners is not a great idea?

Reply:
Matt and I agree, I'll remove the duplicate checking entirely.

Reply:
Removed in 2f73651.